Mathlib/Algebra/Algebra/Field.lean

```diff
@@ -0,0 +1,59 @@
+/-
```

Comment:
IIUC, this just copies the old module docstring. What's mathlib policy here?
(If anybody wants to do archeology: [this mathlib3 blame](https://github.com/leanprover-community/mathlib/blame/master/src/algebra/algebra/basic.lean), starting at line 184, should be helpful.)